src/main/java/homeworkCSV/Data.java

```diff
@@ -0,0 +1,150 @@
+package homeworkCSV;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+public class Data {
+
+    ArrayList<String> dataArray = new ArrayList<>();
+    LinkedList<String> runners = new LinkedList<>();
+    LinkedList<Runner> runnersR = new LinkedList<>();
+    ArrayList<Integer> timeInSeconds = new ArrayList<>();
+    ArrayList<Integer> shootingResults = new ArrayList<>();
+    ArrayList<Integer> totalTime = new ArrayList<>();
```

Comment:
you don't need these lists. Create an Athlete object and store all the details inside

Reply:
The object Runner was created to read data from the CSV file;
I used the lists to take data from the object Runner and do the final calculations. It was the only feasible idea. My idea ;)